packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -474,7 +474,7 @@ function normalizeOutput(
 ) {
   const dir = path.resolve(
     path.dirname(fileURLToPath(import.meta.url)),
-    "../../../",
+    "../../..",
```

Comment:
Let's wait for 23.0.1 before merging the PR, and then revert this line?

Reply:
OK, let me combine the path changes related commits into one so that it would be easier to revert it later.

Reply:
Ok we can then rebase-merge.

Reply:
Rebase done, the first commit now includes all `path.resolve()` related changes.

Reply:
Could you squash all the other commits into one? :)

So that then we merge this PR as two separate commits.